### ISSUE TITLE
Image link centered does not work

### ISSUE THREAD

**Issue by MattSom** [Semantic-Org/Semantic-UI-React#3953]
### Problem description
This is the [very same issue](https://github.com/Semantic-Org/Semantic-UI/issues/4708) with native Semantic-UI.

An `Image` tag with `centered` attribute gets `display: block` formatting making it truly centered. However an `Image` can have an `as='a'` attribute as well, which will wrap the `<img>` tag inside `<a><a/> tags`. 

Example:

```
<Image centered ..... />
<img class="ui small centered image" alt='Disponible en Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/es_badge_web_generic.png'/>

<Image as='a' centered ..... />
<a class="ui small centered image" href=''>
    <img alt='Disponible en Google Play' src='https://play.google.com/intl/en_us/badges/images/generi/es_badge_web_generic.png'/>
</a>
```

### Proposed solution
If the `Image` tag is used as a link while being centered, the anchor tag formatting should be overwritten to this:
```
a.ui.image.centered{
    display: block;
}
```

instead of `display: inline-block`.

**Comment by layershifter** [comment]
We can't fix it there as we don't handle CSS in this repo, but I reported it Fomantic UI team. Let's see how it will go 🚀
Thanks for reporting 👍 

https://github.com/fomantic/Fomantic-UI/issues/1608

**Comment by lubber-de** [comment]
Fixed for Fomantic-UI 2.8.7 by https://github.com/fomantic/Fomantic-UI/pull/1618
